bin/functions

```diff
@@ -108,17 +108,44 @@ function install {
             case ${ASDF_INSTALL_VERSION} in
                 zulu*) mv ./* "${ASDF_INSTALL_PATH}" ;;
                 liberica*) mv ./* "${ASDF_INSTALL_PATH}" ;;
-                *) mv Contents/Home/* "${ASDF_INSTALL_PATH}" ;;
+                *) 
+                    mv Contents/Home/* "${ASDF_INSTALL_PATH}" 
+                    macOS_integration
+                    ;;
             esac ;;
         *) mv ./* "${ASDF_INSTALL_PATH}" ;;
     esac
 }
 
+function uninstall {
+    rm -rf "${ASDF_INSTALL_PATH}"
+    if [ -z ${ASDF_INSTALL_VERSION+x} ]; then 
```

Comment:
We don't want to call `remove_macOS_integration` if `ASDF_INSTALL_VERSION` is set but empty.
It would try to delete `/Library/Java/JavaVirtualMachines/` in that case.
```suggestion
    if [ -z "${ASDF_INSTALL_VERSION}" ]; then 
```

Reply:
You are totally right.